xonsh/environ.py

```diff
@@ -84,6 +86,7 @@ def lc_converter(val):
     'PATHEXT': (is_env_path, str_to_env_path, env_path_to_str),
     'RAISE_SUBPROC_ERROR': (is_bool, to_bool, bool_to_str),
     'RIGHT_PROMPT': (is_string, ensure_string, ensure_string),
+    'DYNAMIC_CWD_WIDTH': (is_string, ensure_string, ensure_string),
```

Comment:
whoops.  never mind -- i see now that you're checking whether it's a percentage or not